apps/gimbtonomy/src/main.cpp

```diff
@@ -16,7 +16,7 @@
 
 #define DEBUG
 
-#define ACK_FLAG (1UL << 0)
+constexpr unsigned long ACK_FLAG(1UL << 0);
```

Comment:
Is there a point of the shift 0 here?

Reply:
It was there from before, I'm assuming for clarity/readibility